macros/schema_tests/expression_is_true.sql

```diff
@@ -7,16 +7,18 @@
 {% macro default__test_expression_is_true(model, expression, column_name, condition) %}
 
 with meet_condition as (
-    select * from {{ model }} where {{ condition }}
+    select
+      *,
+      {% if column_name is none %}
+      {{ expression }}
+      {%- else %}
+      {{ column_name }} {{ expression }}
+      {%- endif %}
+      as _test_expression_passed
```

Comment:
The one thing I'm a bit nervous about here is a field name collision.

A possible solution is to grab the column names from the given model and throw a compiler message if _test_expression_passed is one of the field names.

However, that seems a bit contrived to me. Another possibility is just make the field name more constructed (ie. maybe encode the expression in it etc). Would like to hear peoples thoughts :)

Reply:
I decided to catch this exception in: https://github.com/dbt-labs/dbt-utils/pull/507/commits/68dc34114e6636ee23495e72bed893c002a57ac4.

Let me know if its a bit expensive to use get columns in relation to catch this error, or if people should be left to fend for themselves when seeing "ambiguous column name" errors.

Reply:
It's a fair concern! My gut feeling is that this is a reasonably obscure column name. Perhaps you could include the package name:
```suggestion
      as _dbt_utils_test_expression_passed
```

I don't think it's necessary to protect against the ambiguous column name, just selecting * should be OK.

Reply:
Will do & i'll remove the get column in relation check.